io/LasWriter.cpp

```diff
@@ -485,6 +488,8 @@ void LasWriter::addSpatialRefVlrs()
     deleteVlr(las::TransformUserId, las::GeotiffAsciiRecordId);
     deleteVlr(las::TransformUserId, las::WktRecordId);
     deleteVlr(las::LiblasUserId, las::WktRecordId);
+    // deleteVlr(las::TransformUserId, las::PdalWkt2RecordId);
```

Comment:
lint to remove

Reply:
Done.
I was not sure about this. If I delete those new Vlrs the same way as las::WktRecordId, I cannot run the test that writes different content on each to checks the values properly. The user couldn't neither write something different like an Engineering in wkt1 and a derivedprojected in Wkt2.